content/docs/intro/console/collaboration/auditing.md

```diff
@@ -71,3 +71,121 @@ curl \
 ```
 
 > _Note: Substitute `${org}`, `${user}`,  and `${time}` for your actual values - e.g. `org`, `username`, and `1583460637`._
+
+### Supported Audit Log Formats
+
+We currently support two formats for exporting audit logs through the API. The two formats we currently support
+are CSV and CEF. These formats can be specified by appending the `format` query parameter as follows, `format=csv`
+or `format=cef`.
```

Comment:
```suggestion
Pulumi supports multiple formats for exporting audit logs. These formats can be specified by appending the `format` query parameter, for example, `format=csv`or `format=cef`.
```

Reply:
Thanks, that sounds a million times better!